frontend2/src/api/loaders/myTeamLoader.ts

```diff
@@ -16,5 +22,39 @@ export const myTeamLoader =
       queryFn: async () => await myTeamFactory.queryFn({ episodeId }),
     });
 
+    // Win/loss/tie
+    const teamDataCached = queryClient.getQueryData<TeamPrivate>(
+      buildKey(myTeamFactory.queryKey, { episodeId }),
+    );
+
+    if (!isNil(teamDataCached)) {
+      void queryClient.ensureQueryData({
+        queryKey: buildKey(scrimmagingRecordFactory.queryKey, {
+          episodeId,
+          teamId: teamDataCached.id,
+          scrimmageType:
+            CompeteMatchScrimmagingRecordRetrieveScrimmageTypeEnum.Ranked,
+        }),
+      });
+
+      void queryClient.ensureQueryData({
+        queryKey: buildKey(scrimmagingRecordFactory.queryKey, {
+          episodeId,
+          teamId: teamDataCached.id,
+          scrimmageType:
+            CompeteMatchScrimmagingRecordRetrieveScrimmageTypeEnum.Unranked,
+        }),
+      });
+
+      void queryClient.ensureQueryData({
+        queryKey: buildKey(scrimmagingRecordFactory.queryKey, {
+          episodeId,
+          teamId: teamDataCached.id,
+          scrimmageType:
+            CompeteMatchScrimmagingRecordRetrieveScrimmageTypeEnum.All,
+        }),
+      });
+    }
+
```

Comment:
maybe we can reduce the repetitiveness here too? the main thing that's different is the scrimmage type

Reply:
I reduced it by only ensuring the scrimmage type that is being used on that page :)